content/v2.0/query-data/flux/sql.md

```diff
@@ -154,6 +155,78 @@ Use the variable to manipulate queries in your dashboards.
 
 ---
 
+## Use secrets to populate connection credentials
+If your SQL database requires authentication, use [InfluxDB secrets](/v2.0/security/secrets/)
+to store and populate connection credentials.
+By default, InfluxDB base64-encodes and stores secrets in its internal key-value store, BoltDB.
+For added security, [Vault](https://www.vaultproject.io/).
+
+_For information about the **InfluxDB Vault integration**, see [Store secrets in Vault](/v2.0/security/secrets/use-vault/)._
+
+### Store your database credentials as secrets
+Use the [InfluxDB API](/v2.0/reference/api/) or the [`influx` CLI](/v2.0/reference/cli/influx/secret/)
+to store your database credentials as secrets.
+
+{{< tabs-wrapper >}}
+{{% tabs %}}
+[InfluxDB API](#)
+[influx CLI](#)
+{{% /tabs %}}
+{{% tab-content %}}
+```sh
+curl -X PATCH http://localhost:9999/api/v2/orgs/<org-id>/secrets \
+  -H 'Authorization: Token YOURAUTHTOKEN' \
+  -H 'Content-type: application/json' \
+  -d '{
+  "POSTGRES_HOST": "http://example.com",
+  "POSTGRES_USER": "example-username",
+  "POSTGRES_PASS": "example-password"
+}'
+```
+
+_**Helpful links**_  
+[View your organization ID](/v2.0/organizations/view-orgs/#view-your-organization-id)  
+[View your authentication token](/v2.0/security/tokens/view-tokens/)
```

Comment:
To store secrets, you need your authentication token and organization ID:
[View...